client/apollo/react/src/List/ClickItem/components/ClickItemPrefixCommon.tsx

```diff
@@ -23,20 +23,21 @@ export const ClickItemPrefixCommon = ({
   IconComponent,
   BasePictureComponent,
 }: ClickItemPrefixCommonProps) => {
+  if (!(Boolean(basePictureProps) && variant === "agent") && !icon) {
+    return null;
+  }
+
   if (Boolean(basePictureProps) && variant === "agent") {
     return <BasePictureComponent {...basePictureProps} />;
   }
```

Comment:
Cette condition serait plus simple non ?

```suggestion
  if (Boolean(basePictureProps) && variant === "agent") {
    return <BasePictureComponent {...basePictureProps} />;
  }

  if (!icon) {
    return null;
  }

```

Reply:
oui, mais par pratique on
 renvoyer null d'abord si les conditions ne sont pas remplies

Reply:
Je suis d'accord mais ça duplique une condition.
@samuel-gomez tu en penses quoi ?

Reply:
selon moi on renvoie le null en premier